docs/ref/settings.txt

```diff
@@ -2453,6 +2453,11 @@ all caps and likely starting with ``HTTP_``. (Remember, Django automatically
 adds ``'HTTP_'`` to the start of x-header names before making the header
 available in ``request.META``.)
 
+Additionally, depending on vendors, this header may be provided as 
```

Comment:
Hi @felixxm proposed docs changes are here. 

The ticket additionally mentions https://docs.djangoproject.com/en/4.0/ref/request-response/#django.http.HttpRequest.is_secure but I'm unsure if it really needs any updates since the functionality of the function itself hasn't changed. 

Guidance appreciated